### PR TITLE
[image_picker] Implement methods for picking both images and videos simultaneously.

### PR DESCRIPTION
**Pull request by BeMacized**
This PR implements ~~both `pickImageOrVideo` and `pickMultiImageAndVideo`~~ `pickMedia` on both Android and iOS.

This PR is still in draft mode due ~~to the lack of unit tests, as well as~~ the dependency on the changes to the platform interface: #4675. ~~I will be adding tests later.~~

Relevant issue:
- flutter/flutter#89159

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

File: packages/image_picker/image_picker/ios/Classes/FLTPHPickerSaveItemToPathOperation.m [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+                                                                image:localImage
+                                                             maxWidth:self.maxWidth
+                                                            maxHeight:self.maxHeight
+                                                         imageQuality:self.desiredImageQuality];
+                             [self completeOperationWithPath:savedPath];
+                           }];
+          }
+        }
+      }];
+}
```
Comment (BeMacized) [inline packages/image_picker/image_picker/ios/Classes/FLTPHPickerSaveItemToPathOperation.m:118]: This method was already in here, it's simply moved over from `FLTPHPickerSaveImageToPathOperation`.

**Comment by Hixie** [comment]
It sounds like this is obsolete with https://github.com/flutter/plugins/pull/4675 now, so I'm going to close this PR. Please don't hesitate to submit it again if I'm wrong about this.

**Comment by Hixie** [comment]
@BeMacized Is this PR read for review?

**Comment by BeMacized** [comment]
No, it's not (also because the changes to the platform interface have not been merged yet). 

It will be superceded by two new PRs that I'll submit in a moment. Closing.

**Pull request closed without merging**